arm-recoveryservicessiterecovery/2016-08-10/swagger/service.json

```diff
@@ -6855,7 +6858,7 @@
       "description": "Input details specific to fabrics during Network Mapping.",
       "type": "object",
       "properties": {
-        "InstanceType": {
+        "instanceType": {
```

Comment:
Did the service change or is this update to the swagger making it match the service?